NetflixRouletteWebpack/src/components/error-page/ErrorPage.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useRouteError } from "react-router-dom";
+import "./ErrorPage.scss";
+
+export default function ErrorPage() {
+  const error = useRouteError();
+  console.error(error);
+  console.log(error);
+  console.log("hey");
```

Comment:
@Marysosh Do we really need to log "hey"?

Reply:
Of course we shouldn't😌 Removed this